src/breadcrumbs/__tests__/breadcrumbs.test.js

```diff
@@ -38,6 +38,19 @@ describe('Breadcrumbs', () => {
     ).toMatchSnapshot();
   });
 
+  it('displays trailing separator', () => {
```

Comment:
can we add a visual regression test instead? something like this: https://github.com/uber/baseweb/blob/master/src/breadcrumbs/__tests__/breadcrumbs.scenario.js

Reply:
You bet